<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgoraLoginWidget</class>
 <widget class="QWidget" name="AgoraLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>532</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset theme=":/image/resource/logo-双师@2x.png">
    <normaloff>C:/Users/LC/.designer/backup</normaloff>C:/Users/LC/.designer/backup</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AgoraLoginWidget{
border-radius: 16px;
border: 20px solid #DBDBEA;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="QSubLoginWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#QSubLoginWidget{
	background-color: rgb(255, 255, 255);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="TitleWidget" native="true">
          <property name="styleSheet">
           <string notr="true">#TitleWidget{
background-image:url(&quot;:/image/resource/pic-blue@2x.png&quot;);
background-repeat:no-repeat;
background-position:center;

}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,5,2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QWidget" name="top_widget" native="true">
             <property name="styleSheet">
              <string notr="true">
#top_widget{
border-top-left-radius:10px;
border-top-right-radius:10px;
}

</string>
             </property>
             <layout class="QHBoxLayout" name="TitleLayout">
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="SettingPushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">#SettingPushButton{
border-image:url(&quot;:/image/resource/icon-setting@2x.png&quot;)
}

#SettingPushButton:hover{
border-image:url(&quot;:/image/resource/icon-setting-hover@2x.png&quot;)
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ExitPushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">#ExitPushButton{
border-image:url(&quot;:/image/resource/icon-close-white@2x.png&quot;)
}

#ExitPushButton:hover{
border-image:url(&quot;:/image/resource/icon-close-white-hover@2x.png&quot;)
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="top_center_widget" stretch="3,2,3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="IconLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image:url(&quot;:/image/resource/logo-dual-teacher@2x.png&quot;)

</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="TitleLable">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white</string>
             </property>
             <property name="text">
              <string>声网双师课堂</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="bottom" native="true">
        <property name="styleSheet">
         <string notr="true">#bottom{
background-color: rgb(255, 255, 255);
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
border-bottom:0.5px solid #DBDBEA;
border-right:0.5px solid #DBDBEA;
border-left:0.5px solid #DBDBEA;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="2,1,2,2,1,2">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QWidget" name="RoomWidget" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="RoomIconLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid #E5E6E7;
border-top-left-radius: 6px;
border-bottom-left-radius:6px;
border-right:none;
background-image:url(&quot;:/image/resource/icon-classroom-name.png&quot;);
background-repeat: none;
background-position: center;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="RoomNameLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid #E5E6E7;
border-top-right-radius: 6px;
border-bottom-right-radius:6px;
border-left:none;
background-repeat: none;
background-position: center;
</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>课堂号</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="RoomNameLabel">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="UserWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="UserIconLabel">
              <property name="styleSheet">
               <string notr="true">border: 1px solid #E5E6E7;
border-top-left-radius: 6px;
border-bottom-left-radius:6px;
border-right:none;
background-image:url(&quot;:/image/resource/icon-username.png&quot;);
background-repeat: none;
background-position: center;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="UserNameLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid #E5E6E7;
border-top-right-radius: 6px;
border-bottom-right-radius:6px;
border-left:none;
background-repeat: none;
background-position: center;
</string>
              </property>
              <property name="placeholderText">
               <string>昵称</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
            <property name="spacing">
             <number>190</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="TeachRadioButton">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton:unchecked {
	color: #8a8a9a;
}

QRadioButton::indicator:unchecked {
    width:                  10px;
    height:                 10px;
    border-radius:          6px;
    background-color:       #fff;
    border:                 1px solid #8a8a9a;
}

QRadioButton::indicator:checked {
    width:                  5px;
    height:                 5px;
    border-radius:          6px;
    background-color:       #fff;
    border:                 4px solid #0073ff;
}</string>
              </property>
              <property name="text">
               <string>主讲教室</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ListenRadioButton">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton:unchecked {
	color: #8a8a9a;
}

QRadioButton::indicator:unchecked {
    width:                  10px;
    height:                 10px;
    border-radius:          6px;
    background-color:       #fff;
    border:                 1px solid #8a8a9a;
}

QRadioButton::indicator:checked {
    width:                  5px;
    height:                 5px;
    border-radius:          6px;
    background-color:       #fff;
    border:                 4px solid #0073ff;
}</string>
              </property>
              <property name="text">
               <string>听课教室</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TipsLabel">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="LoginPushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color:#0073FF;
border-radius:6px;
color:white
}

QPushButton:hover{
background-color:#3590FF;
border-radius:6px;
color:white
}

QPushButton:pressed{
background-color: #0E62C8;
border-radius:6px;
color:white
}</string>
           </property>
           <property name="text">
            <string>登 录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../agora_edu_demo.qrc"/>
 </resources>
 <connections/>
</ui>
